modelchecking/ltsmin/src/main/java/net/automatalib/modelcheckers/ltsmin/AbstractLTSmin.java

```diff
@@ -177,6 +178,27 @@ public Function<String, I> getString2Input() {
             throw new ModelCheckingException(ioe);
         }
 
+        final File ltlFile;
+
+        try {
+            // create a file that will contain the LTL of the specification
```

Comment:
Could you add a small comment as to why we are using a file? Basically just a short summary of your PR that long LTL formulae cause issues as parameter